server/ormconfig.ts

```diff
@@ -35,4 +37,5 @@ export const ormConfig: PostgresConnectionOptions = {
   namingStrategy: new SnakeNamingStrategy(),
   synchronize: (process.env.NODE_ENV === 'development' && process.env.SYNC_DATABASE === 'true'),
   logging: false,
+  // logging: true,
```

Comment:
This probably just slipped through. We should actually make it something like `logging: process.env.TYPEORM_LOGGING ?? false` so that this doesn't happen in the future. Feel free to add that if you'd like.